### DISCUSSION TITLE
Can this plugin work with WebGL as well?

### DISCUSSION THREAD

**Discussion by UsmanHexaExperts** [iBicha/UnityYoutubePlayer#131] [General]
Are any specific limitations or configurations required to ensure compatibility with WebGL for this plugin? Has anyone successfully implemented it in a WebGL project, and if so, are there any performance concerns or best practices to follow?

**Comment by iBicha** [comment]
Yes, it works on webgl. But be aware that the video data is streamed through the server, so it is bandwidth intensive.
